visual-tests/src/test-fixtures/indicators/macd.js

```diff
@@ -0,0 +1,50 @@
+(function(d3, fc) {
+    'use strict';
+
+    var data = fc.dataGenerator()
+        .startDate(new Date(2014, 1, 1))
+        .filter(function() { return true; })
+        (100);
+
+    var width = 600, height = 250;
+
+    var container = d3.select('#macd')
+        .append('svg')
+        .attr('width', width)
+        .attr('height', height);
+
+    // Create scale for x axis
+    var dateScale = fc.scale.dateTime()
+        .domain(fc.utilities.extent(data, 'date'))
+        .range([0, width])
+        .nice();
+
+    // Create scale for y axis
+    var priceScale = d3.scale.linear()
+        .range([height, 0])
+        .nice();
+
+    // Create the macd algorithm
+    var macdAlgo = fc.indicators.algorithms.macd()
+        .fastPeriod(4)
+        .slowPeriod(10)
+        .signalPeriod(5)
+        .value(function(d) { return d.open; });
+    macdAlgo(data);
+
+    // compute the extents
```

Comment:
This extent calculation looks funky but I think that’s only because the scale is misnamed?
Also, could be simplified to something like priceScale.domain(d3.max(extent, Math.abs))?